github/reactions.go

```diff
@@ -256,6 +256,101 @@ func (s ReactionsService) CreatePullRequestCommentReaction(ctx context.Context,
 	return m, resp, nil
 }
 
+// ListTeamDiscussionReactions lists the reactions for a team discussion.
+//
+// GitHub API docs: https://developer.github.com/v3/reactions/#list-reactions-for-a-team-discussion
+func (s *ReactionsService) ListTeamDiscussionReactions(ctx context.Context, teamID, discussionNumber int64, opt *ListOptions) ([]*Reaction, *Response, error) {
```

Comment:
I'm thinking that `teamID` (and all IDs on GitHub should remain an `int64`) but that `discussionNumber` and `commentNumber` (reading through the GitHub Developer API docs... it doesn't show up here but probably does below) should be simply `int`.

I think having the discussion and comment number be `int` clues the user of this package into the fact that these are typically small indices and are never expected to be even in the hundreds. I've seen a few GitHub issues go viral before and even then I don't think the indices grew more than a few thousand.

I'd like to hear what @juliaferraioli thinks about this before we make a change, though, so hold off on this until we hear from her.

Reply:
Got it :+1: .

Reply:
Hmmm, I like your reasoning, Glenn. I think it's a nice subtle indication, so I concur.

Reply:
OK, cool.  Thank you, @juliaferraioli!

Let's go ahead with the idea, and make these `*Number` fields `int`s.